test/kwtool_web/controllers/home_controller_test.exs

```diff
@@ -1,15 +1,15 @@
 defmodule KwtoolWeb.HomeControllerTest do
   use KwtoolWeb.ConnCase, async: true
 
-  alias Kwtool.Accounts.Guardian
+  import KwtoolWeb.Support.ConnHelper
```

Comment:
We can import this in `KwtoolWeb.ConnCase`, so that it is available to all controller test 💡

Reply:
updated.